fluss-lake/fluss-lake-paimon/src/test/java/com/alibaba/fluss/lake/paimon/tiering/PaimonTieringTest.java

```diff
@@ -261,25 +261,28 @@ private Tuple2<List<LogRecord>, List<LogRecord>> genPrimaryKeyTableRecords(
 
         // gen +I, -U, +U
         rows = genKvRow(partition, bucket, 1, 4, 7);
+        long currentTime = System.currentTimeMillis();
         writtenLogRecords.addAll(
                 Arrays.asList(
                         toRecord(++offset, rows.get(0), INSERT),
                         toRecord(++offset, rows.get(1), UPDATE_BEFORE),
-                        toRecord(++offset, rows.get(2), UPDATE_AFTER)));
-        expectLogRecords.add(toRecord(offset, rows.get(2), UPDATE_AFTER));
+                        toRecord(++offset, currentTime, rows.get(2), UPDATE_AFTER)));
+        expectLogRecords.add(toRecord(offset, currentTime, rows.get(2), UPDATE_AFTER));
```

Comment:
Why not use `writtenLogRecords.get(writtenLogRecords.size() - 1)` to get the written record?